lib/Cro/HTTP/Request.pm6

```diff
@@ -5,7 +5,7 @@ use Cro::HTTP::BodyParserSelectors;
 use Cro::HTTP::BodySerializerSelectors;
 use Cro::HTTP::Message;
 use Cro::TLS;
-use Cro::Uri::HTTP;
+use Cro::Iri::HTTP;
```

Comment:
Given this is the HTTP request object the HTTP protocol rules apply, so I think this should probably be URI.